node/resolvers/Mutations/Organizations.ts

```diff
@@ -62,6 +68,147 @@ const createUserAndAttachToOrganization = async ({
     })
 }
 
+const createOrganization = async (
+  _: void,
+  {
+    id,
+    name,
+    tradeName,
+    paymentTerms,
+    priceTables,
+    salesChannel,
+    sellers,
+    customFields,
+  }: OrganizationInput,
+  ctx: Context
+): Promise<{
+  Id: string
+  Href: string
+  DocumentId: string
+}> => {
+  const {
+    clients: { masterdata },
+  } = ctx
+
+  const organization = {
+    id,
```

Comment:
I see the id is set here from the `OrganizationInput`. But what happens here if the id is empty? i.e. if it was called from the `createOrganization` (without ID)? Will it create a document with id = "" or master data will give an id to the document?

Reply:
The masterdata never creates the empty ID, in my tests, if it is undefined, null or empty, the ID is generated. Do you think it is necessary to validate the ID and doesn't create if not informed?